Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity_test.py

```diff
@@ -122,3 +122,26 @@ def test_params_to_filter(severity, resolution_status, expected):
     from MicrosoftCloudAppSecurity import params_to_filter
     res = params_to_filter(severity, resolution_status)
     assert res == expected
+
+
+def test_alerts_to_incidents_and_fetch_start_from(requests_mock):
+    """
+    Given:
+        `getLastRun` which holds `last_fetch` and `last_fetch_id`.
+    When:
+        There are two incidents to fetch, That one of them we had already fetched the previous time.
+    Then:
+        We only fetched the one that does not exist in his system.
+    """
+    from MicrosoftCloudAppSecurity import alerts_to_incidents_and_fetch_start_from
+    incidents = get_fetch_data()
+    requests_mock.get('https://demistodev.eu2.portal.cloudappsecurity.com/api/v1/alerts/',
+                      json=incidents["incidents"])
+    res_incidents, fetch_start_time, new_last_fetch_id = \
+        alerts_to_incidents_and_fetch_start_from(incidents["incidents"], 1602771392519, {"last_fetch": 1603365903000,
+                                                 "last_fetch_id": "5f919e55b0703c2f5a23d9d7"})
+    assert fetch_start_time == 1603378041000
+    assert new_last_fetch_id == "5f919e55b0703c2f5a23d9d7"
+    assert res_incidents == [{'name': 'block0', 'occurred': '2020-10-22T17:47:21Z',
+                              'rawJSON': '{"_id": "5f919e55b0703c2f5a23d9d8", "timestamp": 1603378041000, '
```

Comment:
Could you please create a build that fails due to this UT (without your fix), than add the fix and show that the build passes?

Reply:
done